### ISSUE TITLE
Person thumbnails sometimes missing image alt text

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/tiamat-theme#1704]
On a people list page, some persons thumbnails have "This person has no photo" for alt text even if the person does have an image and alt text has been added.

For example, on https://www.colorado.edu/digital-accessibility/people, three people have the alt text on their photo while the others do not.

`<img src="https://www.colorado.edu/digital-accessibility/sites/default/files/styles/focal_image_square/public/people/bq7a1888_original.jpg?h=0a2fd2a0&amp;itok=hB7Ofw5G" alt="This person has no photo">`
